src/main/scala/com/monsanto/arch/cloudformation/model/resource/EC2.scala

```diff
@@ -167,22 +168,22 @@ object `AWS::EC2::Route` extends DefaultJsonProtocol {
     def read(json: JsValue) = ???
```

Comment:
I know that you didn't write this, but perhaps you know why there's a `???` here?